tests/instances.spec.ts

```diff
@@ -351,3 +351,36 @@ test("Export and Upload an instance backup", async ({ page }) => {
   await page.waitForSelector(`text=Created instance ${instance}-1`);
   await deleteInstance(page, `${instance}-1`);
 });
+
+test("Create instance from external instance file", async ({
+  page,
+  lxdVersion,
+}) => {
+  test.skip(
+    lxdVersion === "5.0-edge",
+    "External instance file conversion is only supported in newer versions of LXD",
+  );
+
+  test.skip(
+    Boolean(!process.env.CI),
+    "Skipping test locally for external instance file creation",
+  );
+
+  const INSTANCE_FILE = "ubuntu-minimal.qcow2";
+  await page.goto("/ui/");
+  await page.getByRole("button", { name: "Create instance" }).click();
+  await page.getByRole("button", { name: "Upload instance file" }).click();
+  await page.getByText("External format (.qcow2, .").click();
+  await page
+    .getByRole("textbox", { name: "External format (.qcow2, ." })
+    .setInputFiles(INSTANCE_FILE);
+  const instanceName = "ubuntu-minimal-import";
```

Comment:
This should be randomized to avoid failures on re-running this test and to allow parallel runs.

Reply:
This tests that the instance name is automatically set to the file name + `-import`. The file is downloaded in CI before the e2e tests and have a fixed name, so I don't think the test will fail?

Reply:
My point is: the instance should have a randomized name. Otherwise, retrying this test, if it fails in CI, might cause trouble. I.e. if a test run times out and doesn't finish deleting the instance, then the retry will fail, because it can't create a 2nd instance with the same name.

Reply:
I see, will update the test to input random instance name instead